acnportal/acnsim/network/tests/test_current.py

```diff
@@ -15,24 +15,28 @@ def test_init_loads_dict_input(self):
         pd.testing.assert_series_equal(
             self.current,
             pd.Series([0.25, 0.50, -0.25], index=["PS-001", "PS-002", "PS-003"]),
```

Comment:
The Current class can cast a pd Series object to a Current object if the Series is given as an input to the Current constructor. I think it makes sense to cast the Series to a Current so we can keep the implicit type checking.

Reply:
I pushed a new commit with the proposed solution.